api/envoy/data/accesslog/v2/accesslog.proto

```diff
@@ -54,6 +56,15 @@ message HTTPAccessLogEntry {
   HTTPResponseProperties response = 4;
 }
 
+// Defines fields for a connection
+message ConnectionProperties {
+  // Number of bytes received from downstream.
+  uint64 bytes_received = 1;
+
+  // Number of bytes sent to downstream.
+  uint64 bytes_sent = 2;
+}
```

Comment:
Have we thought at all about incremental logging for long-lived TCP streams? I realize this is also an issue for long-lived HTTP sessions, just curious what the long-term direction is going to be here.

Reply:
Istio would find this quite useful. Currently, we run a timer and report on connection open/close/timer-firing. I think this might need a change in the filter API though (something like `onRecurrentLog` with a customizable interval). This is also super-useful for bi-di gRPC like xDS.

Reply:
Yes I thought about the incremental logging when I added implementation of this, though that is orthogonal to this PR. And it requires invasive change to `AccessLog` interface and logging points. So I think it worth another issue and should be addressed separately.

Reply:
Can you file some tracking issue for this? Thanks.